### ISSUE TITLE
Document labels used by Okteto pods

### ISSUE THREAD

**Issue by vitobotta** [okteto/okteto#3187]
**Is your feature request related to a problem? Please describe.**
Documentation request

**Describe the solution you'd like**
In my case, I use a wrapper around some K8s tools including Okteto to abstract some complexity, and am also using KEDA autoscaling to scale my app. When I start Okteto, the autoscaler interferes because it automatically starts its own non-Okteto pods, so traffic is then split between Okteto pods and the autoscaler pods, causing problems and confusion. For this reason, I changed my CLI so that before starting Okteto it patches the services so that they send traffic only to the Okteto pods, ignoring the autoscaled pods. So I can work with Okteto just fine. 

This wasn't documented, so I had to figure out that Okteto labels the pod for the main service with the label "interactive.dev.okteto.com" and the additional services (as specified in the manifest) with "detached.dev.okteto.com".

By patching my services to use these labels during an Okteto session, I no longer have problems due to the autoscaling.

**Describe alternatives you've considered**
I couldn't think of an alternative

**Comment by jLopezbarb** [comment]
@RinkiyaKeDad Should we move this issue to the docs repo? Where is the best place to document those labels?

**Comment by RinkiyaKeDad** [comment]
Yes. Let's first create an issue on [github.com/okteto/docs](https://github.com/okteto/docs) and there we can discuss where to document this!
